.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+### What works
+
+### What was not tested
+
+### Test
+Operating system (and linux kernel version): 
+Hardware (GPU/CPU):
+
+### Ready for review
+[ ] Script tested as a regular phoenicis user and working (if you have a problem -> draft and ask for help).
+[ ] json-align and eslint run accorded to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
+[ ] Codacy and travis checked.
```

Comment:
This makes no sense in my opinion. You can't check this before creating the PR. In addition who should then check this the author of the PR or the reviewers?

Reply:
At the first time I had written this, The sentence was "The PR will be reviewed when codacy and travis checks are ok" more or less, because I am a little bit tired to always remind people of checking them.

Reply:
The question for me is whether it is really the task of the issue author to check Codacy and Travis. Yes, both Travis and Codacy need to succeed before the PR can be merged, but still the PR author can't check that before submitting the PR.